pkg/defaults/policies/files/pod_exec.json

```diff
@@ -47,6 +47,15 @@
           "namespace": "openshift-etcd"
         }
       }
+      },
+      {
+      "name": "Don't alert on deployment workspace\\w{16} in namespace openshift-terminal",
+      "deployment": {
+        "name": "workspace\\w{16}",
```

Comment:
@sluetze what names do you intend to cover? Hard to validate regex correctness without a description : )

Reply:
Please put your comment into the PR description so it's easier for posterity to find it.

Reply:
done